webapp/sources/rudder/rudder-core/src/main/scala/com/normation/cfclerk/xmlparsers/CfclerkXmlConstants.scala

```diff
@@ -110,6 +110,7 @@ object CfclerkXmlConstants {
   val VAR_NAME                        = "NAME"
   val VAR_DESCRIPTION                 = "DESCRIPTION"
   val VAR_LONG_DESCRIPTION            = "LONGDESCRIPTION"
+  val VAR_VARIABLE_NAME               = "VARIABLE_NAME"
```

Comment:
I would use `VARIABLENAME` to be consistent with other properties here. Or change for underscore everywhere.